docs/graphql-api.md

```diff
@@ -0,0 +1,184 @@
+---
+title: GraphQL API
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /graphql-api.html
+summary:
+---
+
+## GraphiQL
+
+Explore the GraphQL API at
+
+http://localhost:3000/api/v1/graph/iql
+
+with the “Documentation Explorer” on the right side.
+
+
+![GraphiQL screenshot](https://user-images.githubusercontent.com/815894/28073533-47debff6-6656-11e7-82fb-eb5aebefa350.png)
```

Comment:
We should not include images/screenshots of our interfaces. It would cause a lot more work if we decide to modify the interface in the future.